app/views/mailers/response_mailer/confirmation_email.et3-v1-en.html.erb

```diff
@@ -1,3 +1,4 @@
+<% message.subject = 'Your Response to Employment Tribunal claim online form receipt' %>
```

Comment:
As the message subject is language specific - made sense to set it in the email template